system-addon/lib/TopSitesFeed.jsm

```diff
@@ -22,8 +23,11 @@ const DEFAULT_TOP_SITES = [];
 this.TopSitesFeed = class TopSitesFeed {
   constructor() {
     this.lastUpdated = 0;
+    this._tippyTopProvider = new TippyTopProvider();
   }
   init() {
+    this._tippyTopProvider.init();
```

Comment:
Sorry for the conflict with removing `TopSitesFeed.init` Maybe just call this just after constructing it from `TopSitesFeed.constructor` ?